api/lib/domain/read-models/AllowedCertificationCenterAccess.js

```diff
@@ -69,6 +70,10 @@ class AllowedCertificationCenterAccess {
   isInWhitelist() {
     return features.pixCertifScoBlockedAccessWhitelist.includes(this.externalId.toUpperCase());
   }
+
+  hasSupervisorPortal() {
+    return config.features.endTestScreenRemovalWhiteList.includes(this.id);
```

Comment:
Comme suggéré par Clement:
```suggestion
    return features.endTestScreenRemovalWhiteList.includes(this.id);
```